src/material-luxon-adapter/adapter/luxon-date-adapter.ts

```diff
@@ -71,6 +81,9 @@ export class LuxonDateAdapter extends DateAdapter<LuxonDateTime> {
     this._useUTC = !!options?.useUtc;
     this._firstDayOfWeek = options?.firstDayOfWeek || 0;
     this.setLocale(dateLocale || LuxonDateTime.local().locale);
+
+    this._defaultOutputCalendar = options?.defaultOutputCalendar || 'gregory';
+    LuxonSettings.defaultOutputCalendar = this._defaultOutputCalendar;
```

Comment:
Doing this will affect uses of Luxon outside of Angular Material which we generally try to avoid. Can we do this only when constructing a date instead?

Reply:
Updated the code. Thanks